### PR TITLE
improve default truffle-config.js readability

### PR DESCRIPTION
**Pull request by davidmurdoch**
This PR isn't trying to make it perfect, just a little bit better. :-)

### CODE REVIEW

**Review by haltman-at — APPROVED** [review]
Makes sense to me.  Thanks!

File: packages/core/lib/commands/init/initSource/truffle-config.js [hunk 1]
```diff
@@ -46,34 +46,37 @@ module.exports = {
...
+    //   network_id: 1342,       // Custom network
+    //   gas: 8500000,           // Gas sent with each transaction (default: ~6700000)
+    //   gasPrice: 20000000000,  // 20 gwei (in wei) (default: 100 gwei)
+    //   from: <address>,        // Account to send transactions from (default: accounts[0])
+    //   websocket: true         // Enable EventEmitter interface for web3 (default: false)
     // },
+    //
     // Useful for deploying to a public network.
-    // NB: It's important to wrap the provider as a function.
+    // Note: It's important to wrap the provider as a function to ensure truffle uses a new provider every time.
```
Comment (cds-amal) [inline packages/core/lib/commands/init/initSource/truffle-config.js:61]: Uses a new provider every time is ambiguous. I would rather leave it as the suggestion below and expand the reasoning in our documentation. 

```suggestion
    // Note: It's important to wrap the provider as a function.
```

**Review by cds-amal — CHANGES_REQUESTED** [review]
Oops, missed this.

**Review by eggplantzzz — APPROVED** [review]
I like this.

**Pull request merged**